### DISCUSSION TITLE
How to self union multipolygon layer a GeoDataFrame?

### DISCUSSION THREAD

**Discussion by PhilLandia** [geopandas/geopandas#2905] [Q&A]
I need to perform a union operation inside a single layer. In Qgis there is an option not to select the second layer, but I can't figure out how to do it in geopandas?
Example:
![image](https://github.com/geopandas/geopandas/assets/46634870/57567241-9f20-4447-8609-84ff25b1171c)
from this photo I should get the following result:
![image](https://github.com/geopandas/geopandas/assets/46634870/8048e79d-6ca3-4eb0-8c4f-e56049c01f7f)
Can you write the code of this operation? Something like this: union = gpd.overlay(zone, how='union')

**Comment by martinfleis** [comment]
Can you provide a reproducible example? I am not fully sure about your data structure.

**Reply by PhilLandia** [reply]
Excuse me, what example are you talking about?

**Reply by martinfleis** [reply]
I would need an example like those explained in [this guide](https://matthewrocklin.com/minimal-bug-reports) to be able to reproduce your issue and help you.

**Reply by PhilLandia** [reply]
```
import geopandas as gpd
land_plots = pd.DataFrame(...)
zones = pd.DataFrame(...)
#Perform initial spatial intersection
intersected = gpd.overlay(land_plots, zones, how='intersection')
union = gpd.overlay(intersected,intersected, how='union')
```
This operation does not produce the desired result
